fix: minor information to docs (#290)
aerogear/offix: docs/getting-started.md

sidebar_label: Getting Started
---

> Offix can be used with existing Apollo GraphQL applications as form of the extension to Apollo Client.

Reviewer: ```suggestion
Offix can be used with existing Apollo GraphQL applications as an extension to the Apollo Client.
```

> Client will provide additional methods that will support various offline use cases. 

Reviewer: ```suggestion
The Offix Client provides additional methods that support various offline use cases. 
```

yarn add offix-client
```

> ## Using the Client inside your application

Reviewer: ```suggestion
## Using the Client in your Application
```

> To work with Offix you should create Offix client. 

Reviewer: ```suggestion
To work with Offix, create an `ApolloOfflineClient`
```

const client = new ApolloOfflineClient(config);

// initialise the client

> // Client needs to be installised before any other queries and mutations will happen.

Reviewer: ```suggestion
// The client must be initialised before any queries or mutations are called.
```

## Offix Client Boost

> The `offix-client-boost` is a convenient way to create a client already bundled with all you nead to work with GraphQL. 

Reviewer: ```suggestion
The `offix-client-boost` is a convenient way to create a client already bundled with all you need to work with GraphQL. 
```

> Mainly an cache and [Apollo Links](https://www.apollographql.com/docs/link/) subscriptions and file uploads.

Reviewer: ```suggestion
A cache and [Apollo Links](https://www.apollographql.com/docs/link/) for subscriptions and file uploads are provided.
```

> Offix Boost is recomended if you trying to build your first GraphQL application and want to have seamless experience.

Reviewer: ```suggestion
Offix Boost is recommended if you are trying to build your first GraphQL application and you want to have a seamless experience.
```

const client = await createClient(config)
```

> ## Working with client

Reviewer: ```suggestion
## Working with the Client
```

> Client will offer the same API as Apollo GraphQL client.

Reviewer: ```suggestion
The Offix client offers the same API as the Apollo GraphQL client and supplies additional methods for building offline experiences.
```

> Offix will supply additional methods that help with offline experience.

Reviewer: ```suggestion

```

}
```

## Non Offline realated API

> `ApolloOfflineClient` is and extension of the Apollo GraphQL client and can be used with various web and mobile frameworks.

Reviewer: ```suggestion
`ApolloOfflineClient` is an extension of the Apollo GraphQL client and can be used with various web and mobile frameworks.
```
